dev/src/components/profile/ProfileForm.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { Dispatch, SetStateAction, useMemo } from "react"
+import React, { Dispatch, SetStateAction, useMemo, useState } from "react"
```

Comment:
`useMemo` is no longer used. (DeepSource)... Other DeepSource (anti-pattern) issues are better resolved in a separate PR.

Reply:
Just realized that we can add this to `tsconfig.json` to be notified of such issues:

```
    "noUnusedLocals": true,
    "noUnusedParameters": true
```

Reply:
Yes, please do!